src/file-format-verification/components/ParseErrors.jsx

```diff
@@ -77,9 +77,10 @@ const renderTSErrors = transmittalSheetErrors => {
   )
 }
 
-const renderParseResults = count => {
-  const noErrors = count === 0
+const renderParseResults = (count, errors) => {
```

Comment:
Can we change these names a bit to differentiate between formatting errors and file/http errors? Right now this function reads oddly if you don't expect the distinction.